Jenkinsfile

```diff
@@ -1,153 +1,103 @@
 #!/usr/bin/env groovy
 
-def dockerRegistry = "329802642264.dkr.ecr.eu-west-1.amazonaws.com"
-def nodeImageVersion = "10.16.0"
-def nodeImage = "${dockerRegistry}/bbc-news/node-10-lts:${nodeImageVersion}"
+def nodeImage = "329802642264.dkr.ecr.eu-west-1.amazonaws.com/bbc-news/node-10-lts:10.16.0"
 
-def nodeName
 def slackChannel = "#si_repo-psammead"
-def stageName = "Unknown"
-def gitCommitAuthor = "Unknown"
-def gitCommitMessage = "Unknown"
 
-def getCommitInfo = {
-  gitCommitAuthor = sh(returnStdout: true, script: "git --no-pager show -s --format='%an' ${GIT_COMMIT}").trim()
-  gitCommitMessage = sh(returnStdout: true, script: "git log -1 --pretty=%B").trim()
+def notifySlack(colour, buildStatus, gitCommitAuthor, info, gitCommit, gitCommitMessage, slackChannel) {
+  // call the global slackSend method in Jenkins
+  slackSend(color: colour, message: "*${buildStatus}* on ${env.BRANCH_NAME} [build ${BUILD_DISPLAY_NAME}] \n*Author:* ${gitCommitAuthor} \n*Info:* ${info} \n*Commit Hash* \n${gitCommit} \n*Commit Message* \n${gitCommitMessage}", channel: slackChannel)
 }
 
-def notifySlack(colour, buildStatus, gitCommitAuthor, stageName, gitCommitMessage, slackChannel) {
-  // call the global slackSend method in Jenkins
-  slackSend(color: colour, message: "*${buildStatus}* on ${GIT_BRANCH} [build ${BUILD_DISPLAY_NAME}] \n*Author:* ${gitCommitAuthor} \n*Stage:* ${stageName} \n*Commit Hash* \n${GIT_COMMIT} \n*Commit Message* \n${gitCommitMessage}", channel: slackChannel)
+def cleanUp() {
+  sh 'chmod -R 777 .git'
```

Comment:
i don't understand this method - why change the permissions? and im not sure how it's going to clean up?

Reply:
This is from the talos implementation. According to @dr3 777 permissions are required to jenkins can remove the .git folder ready for the next build. A new issues is being raised to see if we can move away from have to do this.

Reply:
fair enough